### DISCUSSION TITLE
video tutorial

### DISCUSSION THREAD

**Discussion by thunderwin** [uicrooks/shopify-theme-lab#148] [General]
it there any video tutorial  . we don't know where to start

**Comment by sergejcodes** [comment]
No, not specifically for Theme Lab.

To get started you can run down this list:

1. Search for `Shopify CLI tutorials`, since `Theme Lab` is built on top of the `Shopify CLI` a lot of things apply to it. If you understand how the `Shopify CLI` works, it will be easy to work with `Theme Lab`.

2. Setup your Shopify Store with the Theme Lab theme. The Shopify CLI tutorials should help. [Here's also the Installation guide](https://themelab.uicrooks.com/guide/getting-started.html#installation)

3. Familiarize yourself with the Theme Lab specific [tasks/commands](https://themelab.uicrooks.com/guide/getting-started.html#tasks). 

4. Have a look at this [theme project](https://github.com/uicrooks/shopify-foundation-theme) which is built with Theme Lab. For some code examples.

Hope this helps.
